### DISCUSSION TITLE
nethserver based centos no ping other peers

### DISCUSSION THREAD

**Discussion by delascosas** [netbirdio/netbird#370] [Q&A / Support]
ping: sendmsg: Operation not permitted

samebody help me??

**Comment by braginini** [comment]
What is the OS you are running NetBird on?
What is the NetBird version?
Self-hosted or managed version?

**Reply by delascosas** [reply]
managed version at app.netbird.io

**Reply by delascosas** [reply]
in a virtual box vm running nethserver, on a Windows (also peer in the same account) host that can ping to other peers...

**Reply by mlsmaycon** [reply]
Ok thanks for the info @delascosas ,

Could you please run the following commands and share the output with us?

netbird version
ip addr show wt0
ip route show

If you have iptables, please run:
iptables -L -n
iptables -L -n -t nat

One last question, do you have other VPNs running in the same host? If so, have you tested without them?

**Reply by delascosas** [reply]
root@neuralgico ~]# ping 100.127.200.74
PING 100.127.200.74 (100.127.200.74) 56(84) bytes of data.
From 100.127.70.162 icmp_seq=1 Destination Host Unreachable
ping: sendmsg: Operation not permitted
From 100.127.70.162 icmp_seq=2 Destination Host Unreachable
ping: sendmsg: Operation not permitted
From 100.127.70.162 icmp_seq=3 Destination Host Unreachable
ping: sendmsg: Operation not permitted
^C
--- 100.127.200.74 ping statistics ---
3 packets transmitted, 0 received, +3 errors, 100% packet loss, time 2001ms

[root@neuralgico ~]# ip route show
default via 192.168.100.1 dev enp0s3
10.0.2.0/24 dev enp0s8 proto kernel scope link src 10.0.2.110
10.77.0.0/24 dev tunrw proto kernel scope link src 10.77.0.1
10.239.130.0/24 dev tuntest20210619 proto kernel scope link src 10.239.130.1 100.127.0.0/16 dev wt0 proto kernel scope link src 100.127.70.162
172.17.0.0/16 dev docker0 proto kernel scope link src 172.17.0.1
172.28.0.0/16 dev aqua0 proto kernel scope link src 172.28.0.1
192.168.100.0/24 dev enp0s3 proto kernel scope link src 192.168.100.100
[root@neuralgico ~]# netbird version
0.8.0
[root@neuralgico ~]# ip addr show wt0
10: wt0: <POINTOPOINT,MULTICAST,NOARP,UP,LOWER_UP> mtu 1280 qdisc pfifo_fast state UNKNOWN group default qlen 500
    link/none
    inet 100.127.70.162/16 brd 100.127.255.255 scope global wt0
       valid_lft forever preferred_lft forever
    inet6 fe80::ad3a:1aec:d085:dba6/64 scope link flags 800
       valid_lft forever preferred_lft forever
[root@neuralgico ~]#

**Reply by mlsmaycon** [reply]
Ok, windows machine's have firewall blocking incoming traffic by default, can you confirm if the issue also happens from the windows host to the nethserver?

Also, could you please install wireguard-tools (regular or repel repo's) package and run:

sudo wg show

On the nethserver peer?

**Reply by mlsmaycon** [reply]
Hi @delascosas did you solve the issue?
